roundtrip/RoundTrip/Spec.hs

```diff
@@ -118,17 +118,18 @@ roundTripSpec = do
               -- IPC
               resp <- doReq hin herr hout (Req RTPlutusData payload)
               -- Assert response
-              pdResp <-
+              payload' <-
                 response
                   (\err -> assertFailure $ "hs> Wanted ResSuccess got ResError " <> err)
```

Comment:
You could have a helper like `assertFailure' = asssertFailure . ("hs>" <>)` to factor out that part

Reply:
Done